MVP/defaults-build.yaml

```diff
@@ -0,0 +1,74 @@
+- defaults:
+    name: defaults-build
+    description: |
+        Managed by Jenkins Job Builder. Do not edit via web.
+    node: 'jslave-platform-rhel7' #[MVP] please replace this value with jenkins slave name/label of your own.
+    #concurrent: true
+    parameters:
+        - string:
+            name: id
+        - string:
+            name: task_id
+        - string:
+            name: CI_TYPE
+        - string:
+            name: CI_MESSAGE
+        - string:
+            name: tag 
+        - string:
+            name: name
+        - string:
+            name: version
+        - string:
+            name: release
```

Comment:
I assume this is just for the MVP. Since most of the parameters can be queried from brew based on the task_id and we already have the code for it.